content/v2.0/actions/testing.md

```diff
@@ -12,7 +12,7 @@ The examples on this page use [RSpec](http://rspec.info), the test framework ins
 Actions are standalone objects with an interface that's easy to test. You can simply instantiate an action as your object under test and exercise its functionality.
 
 ```ruby
-# spec/app/actions/books/index_spec.rb
+# spec/actions/books/index_spec.rb
```

Comment:
Q: Because we can also test slices, wouldn't we want to change the actual spec folder `spec/actions -> spec/app/actions` instead of changing this path here?

Reply:
Looks like `bundle exec hanami generate action books.index` generates an action spec at `spec/actions/books/index_spec.rb` @swilgosz, so I think let's go with the change as suggested here by @Drowze.